bindings/pyroot/pythonizations/python/ROOT/_numbadeclare.py

```diff
@@ -50,6 +50,23 @@ def get_inner_type(t):
         Get inner typename of a templated C++ typename
         '''
         try:
+            if '<' not in t:
+                # therefore, type must use a shorthand alias
+                typemap = {
```

Comment:
Perhaps this map can be declared once at the decorator function level as `rvec_typemap` and then used inside this helper function.

Reply:
Or just leave it here, since I see the same pattern is followed in other helper functions.